src/metric-engine/src/engine/create.rs

```diff
@@ -178,6 +190,7 @@ impl MetricEngineInner {
                 metadata_region_id,
                 logical_region_id,
                 new_columns,
+                alter_request,
```

Comment:
I think we need to make the call idempotent, and the `alter_request` should return the same content each time, in order for metasrv to take a retry.

Reply:
Maybe we should return the physical table schema to metasrv🥲

Reply:
> Maybe we should return the physical table schema to metasrv🥲

+1